payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionUI.kt

```diff
@@ -109,15 +78,18 @@ internal fun SectionTitle(@StringRes titleText: Int?) {
 @Composable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun SectionCard(
+    modifier: Modifier = Modifier,
+    isSelected: Boolean = false,
     content: @Composable () -> Unit
 ) {
     Card(
-        border = BorderStroke(CardStyle.cardBorderWidth, CardStyle.cardBorderColor),
-        elevation = CardStyle.cardElevation,
+        border = PaymentsTheme.getBorderStroke(isSelected),
+        // todo skyler: this will change when we add shadow configurations.
+        elevation = if (isSelected) 1.5.dp else 0.dp,
+        backgroundColor = PaymentsTheme.colors.colorComponentBackground,
+        modifier = modifier
     ) {
-        Column {
-            content()
-        }
+        content()
```

Comment:
Can you verify this looks correct on the delayed payment method SEPA, specifically the address field?

Reply:
| Master  | This PR |
| ------------- | ------------- |
|![sepaMaster](https://user-images.githubusercontent.com/89166418/157997516-f51a740a-1508-4dbd-a9f4-562a524098a8.png)|![sepaBranch](https://user-images.githubusercontent.com/89166418/157997515-a7df89ab-7eb2-46d8-8c03-74d4d0902a42.png)